src/lib/css-parse.html

```diff
@@ -89,6 +90,21 @@
       return node;
     },
 
+    // conversion of sort unicode escapes with spaces like `\33 ` (and longer) into

```

Comment:
The problem is that CSS allows unicode escapes like `.\33 d-model` (because `.3d-model` seems to be invalid), which requires space after escape sequence in short form (2 hex digits in this example).
It is all good, but current CSS parser in Polymer is simple regexp based one, so space after escape sequence causes problems for it, see https://github.com/Polymer/polymer/issues/2650 for background. Make parser that covers all cases would make everything unnecessary more complex, so I've chosen another way.

This function converts short escape sequence into long (namely `\33 ` into `\000033`, 6 hex digits), which doesn't actually require space after it and thus doesn't cause any problems for Polymer parser, while functioning in exactly the same way (see tests added).